CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change Log for corePKCS11 Library
 
+## v3.5.0 (August 2022)
+* [#156](https://github.com/FreeRTOS/corePKCS11/pull/156) MISRA compliance updates
```

Comment:
```suggestion
* [#156](https://github.com/FreeRTOS/corePKCS11/pull/156) MISRA C:2012 compliance updates
```

Reply:
Added that to the line, also did that in other repos as well.